sw/otbn/crypto/tests/p384_keygen_from_seed_test.exp

```diff
@@ -0,0 +1,3 @@
+# Expected values:
+w0 = 0x0000000000000000000000000000000000000000000000000000000000000000
+w1 = 0x0000000000000000000000000000000000000000000000000000000000000000
```

Comment:
nit: I worry that an all-zero expected result might cause a bug (e.g. the program `ecall`s immediately) to get missed. Perhaps instead of URND the test could use a known value, and then compare to a full expected result? The URND from the OTBN simulator is just hardcoded constants anyway iirc, so it wouldn't affect coverage. Same comment applies to the `p384_boolean_to_arithmetic_test`.

I don't think this should block the PR, so if you don't want to address it here that's fine -- although it would be nice to open an issue so we can track it as a future improvement :slightly_smiling_face:

Reply:
In this case, this would not only be relevant for `p384_keygen_from_seed_test` and `p384_boolean_to_arithmetic_test`, but also for `p384_keygen_test`, `p384_arithmetic_to_boolean_mod_test` and `p384_arithmetic_to_boolean_test`. All these tests expect an all-zero result. 

Therefore, as this affects a couple of tests, I would suggest to open an issue and address this in a separate PR.

Reply:
Actually, it's also the case for `p256_arithmetic_to_boolean_mod_test` and `p384_arithmetic_to_boolean_test`.